### PR TITLE
Fix/docs publishing

### PR DESCRIPTION
**Pull request by InteractiveTimmy**
The changes in this pull request are inclusive of the fixes needed to create and publish storybook documentation.

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -100,15 +88,16 @@ workflows:
...
+            - test
           filters:
             branches:
               only: master
-      - storybook:
+      - release:
           requires:
-            - release
+            - build
+            - storybook
```
Comment (InteractiveTimmy) [inline .circleci/config.yml]: I merged both build and storybook into one set.

File: release.config.js [hunk 2]
```diff
@@ -20,7 +20,7 @@ module.exports = {
     [
       '@semantic-release/git',
       {
-        assets: ['CHANGELOG.md', 'package.json', 'package-lock.json'],
+        assets: ['CHANGELOG.md', 'docs/', 'package.json', 'package-lock.json'],
```
Comment (InteractiveTimmy) [inline release.config.js]: publishing all content in the docs folder in case we add other documentation later.

File: .circleci/config.yml [hunk 3]
```diff
@@ -73,18 +73,6 @@ jobs:
       - run:
           name: Build Storybook
           command: npm run storybook:ci
-      - run:
```
Comment (taymoork2) [inline .circleci/config.yml]: You'll need to store `docs/` to the workspace so that the `release` job can pick it up

Comment (InteractiveTimmy) [reply .circleci/config.yml]: Complete.

**Pull request merged**